aiida/cmdline/commands/cmd_status.py

```diff
@@ -130,7 +130,7 @@ def verdi_status(print_traceback, no_rmq):
         delete_stale_pid_file(client)
         daemon_status = get_daemon_status(client)
 
-        daemon_status = daemon_status.split('\n')[0]  # take only the first line
+        daemon_status = daemon_status.split('\n', maxsplit=1)[0]  # take only the first line
```

Comment:
what does this accomplish?

Reply:
Instead of splitting `"a\nb\nc" in to `["a", "b", "c"]`, it will split only on the first occurrence then return, e.g. `["a", "b\nc"]`, which is more performant if you only need the first item. Same with `rsplit`, if you only need the last item

Reply:
I see, probably a nano-optimization, but I will allow it ;)

Reply:
Well that lucky, because pylint won't 😜